packages/big-design/src/components/Select/Select.tsx

```diff
@@ -465,3 +480,15 @@ const iconColor = <T extends any>(item: SelectOption<T> | SelectAction, isHighli
 
   return 'actionType' in item ? (item.actionType === 'destructive' ? 'danger50' : 'primary') : 'primary';
 };
+
+const descriptionColor = <T extends any>(item: SelectOption<T> | SelectAction, isHighlighted: boolean) => {
+  if (item.disabled) {
+    return 'secondary40';
+  }
+
+  if (!isHighlighted || !('onActionClick' in item)) {
+    return 'secondary60';
+  }
+
+  return 'actionType' in item ? (item.actionType === 'destructive' ? 'danger30' : 'primary30') : 'primary30';
```

Comment:
@valterfatia `danger30` color with `danger10` background-color doesn't meet accessibility color contrast ratios. Is there another option?

Reply:
I have been reviewing that change within the design team and we have agreed to keep secondary60 as the text colour to meet accessibility requirements.